pkg/grub/Dockerfile

```diff
@@ -25,10 +25,10 @@ RUN eve-alpine-deploy.sh
 # list of grub modules that are portable between x86_64 and aarch64
```

Comment:
Typo in the commit message: "minot" -> "minor"

Reply:
Fixed. Thanks.